internal/logger/interface.go

```diff
@@ -0,0 +1,9 @@
+package logger
```

Comment:
Remove the whole logger package and use slog instead https://go.dev/blog/slog or use zerlog as we do everywhere.

Reply:
Okay. The logger wrapper package and interface were removed. Now I use slog instead.